analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RoslynSymbolicExecution.cs

```diff
@@ -185,7 +184,16 @@ private IEnumerable<ExplodedNode> ProcessOperation(ExplodedNode node)
                     yield return node.CreateNext(node.Operation.Parent is null && node.Block.BranchValue != node.Operation.Instance ? postProcessed.State.ResetOperations() : postProcessed.State);
                 }
             }
-            if (ExceptionCandidate(node.Operation) is { } exception && node.Block.EnclosingRegion(ControlFlowRegionKind.Try) is { } tryRegion)
+
+            foreach (var explodedNode in ExceptionSuccessors(node, ExceptionCandidate(node.Operation.Instance)))
```

Comment:
We should rename the `explodedNode`, it's not clear what's the difference between `explodedNode` and `node`, both are exploded. Maybe like this?
```suggestion
            foreach (var successor in ExceptionSuccessors(node, ExceptionCandidate(node.Operation.Instance)))
```